index.js

```diff
@@ -189,3 +253,7 @@ module.exports.addAlias = addAlias
 module.exports.addAliases = addAliases
 module.exports.isPathMatchesAlias = isPathMatchesAlias
 module.exports.reset = reset
+
+module.exports._loadConfig = loadConfig
+module.exports._loadConfigFile = loadConfigFile
+module.exports._loadPackageJSONFile = loadPackageJSONFile
```

Comment:
I would not export those three private functions for testing ; I'd rather test through the public API.

Reply:
Yeah, sounds reasonable. I'll instead test different cases with module to cover potential things.